pkg/controller/controller.go

```diff
@@ -108,10 +108,10 @@ func (c *Controller) Start(stopCh <-chan struct{}) error {
 			return fmt.Errorf("fail to start ringbuf reader: %v", err)
 		}
 	}
-	c.client = NewXdsClient(c.mode, c.bpfAdsObj, c.bpfWorkloadObj, c.enableAccesslog)
+	c.client = NewXdsClient(c.mode, c.bpfAdsObj, c.bpfWorkloadObj, c.enableAccesslog, c.enablePerfMonitor)
```

Comment:
We should add the ability of kmeshctl to control the bpf perf monitor

Reply:
SGTM